code/modules/clothing/chameleon.dm

```diff
@@ -212,7 +212,7 @@
 		thing.update_slot_icon()
 	UpdateButtonIcon()
 
-/datum/action/item_action/chameleon/change/proc/update_item(obj/item/picked_item)
+/datum/action/item_action/chameleon/change/proc/update_item(obj/item/picked_item, obj/item/target = target) //yogs -- add support for cham hardsuits
```

Comment:
```suggestion
/datum/action/item_action/chameleon/change/proc/update_item(obj/item/picked_item) //yogs -- add support for cham hardsuits
```
The second argument causes a runtime

Reply:
... what's the runtime? also there's a reason that argument was added.

Reply:
fixed. it needed to be `target = src.target`